massa-execution-worker/src/tests/universe.rs

```diff
@@ -4,10 +4,11 @@ use std::{
     sync::Arc,
 };
 
-#[cfg(feature = "file_storage_backend")]
+#[cfg(all(feature = "file_storage_backend", not(feature = "db_storage_backend")))]
```

Comment:
maybe
```rust
#[cfg(all(feature = "db_storage_backend", feature = "db_storage_backend"))]
compile_error!("Features db_storage_backend and file_storage_backend are mutually exclusive");
#[cfg(feature = "file_storage_backend")]
use crate::storage_backend::FileStorageBackend;
#[cfg(feature = "db_storage_backend")]
use crate::storage_backend::RocksDBStorageBackend;
```
is a bit clearer?

Reply:
But in that case the CI won't work because of the all features flag.
It's possible to set manually all the features to exclude one of them, but If it's possible to avoid I'd prefer :)

Reply:
oh yes my bad
so simply forget my comment